src/features/donations/components/paymentMethods/CardPayments.tsx

```diff
@@ -240,6 +244,14 @@ function CardPayments({
           <div onClick={handleSubmit} className={styles.actionButtonsContainerCenter}>
             <AnimatedButton className={styles.continueButton} id='donateContinueButton'>
               {t('common:donate')}
+              <div className={styles.totalCost}
+              style={{ color: styles.light, fontSize: "14px" }}>
+              {getFormatedCurrency(
+                    i18n.language,
+                    currency,
+                    treeCount * treeCost
+                  )}
```

Comment:
This only adds the amount to the enabled donate button (not the disabled version) and only for credit card payment - is that intended?

<img width="303" alt="Bildschirmfoto 2021-04-09 um 10 29 46" src="https://user-images.githubusercontent.com/1532418/114152588-bfbf8380-991e-11eb-9821-b53db8f7539f.png"><img width="306" alt="Bildschirmfoto 2021-04-09 um 10 30 04" src="https://user-images.githubusercontent.com/1532418/114152592-c0f0b080-991e-11eb-8955-01ecbb0848b0.png">

Reply:
@rishsingh07  please add at other places also

Reply:
@harshvitra @rishsingh07 I don't think this makes sense for `Continue with Sofort`, or `Continue with Giropay` buttons.